templates/form/form_theme.html.twig

```diff
@@ -0,0 +1,188 @@
+{% extends "form_div_layout.html.twig" %}
+
+{%- block form_row -%}
```

Comment:
Why are you using a dash in `{%-` and `-%}` ?

Reply:
it allows to delete spaces at the beginning and end. Here are the links of the doc if you want to check :[https://twig.symfony.com/doc/2.x/templates.html#whitespace-control](url)